iina/Utility.swift

```diff
@@ -461,18 +461,9 @@ class Utility {
     guard let contents = try? FileManager.default.contentsOfDirectory(
       at: folder,
       includingPropertiesForKeys: [.creationDateKey],
-      options: .skipsSubdirectoryDescendants) else { return nil }
-
-    var latestDate = Date.distantPast
-    var latestFile: URL = contents[0]
-
-    for file in contents {
-      if let date = try? file.resourceValues(forKeys: [.creationDateKey]).creationDate, date > latestDate {
-        latestDate = date
-        latestFile = file
-      }
-    }
-    return latestFile
+      options: .skipsSubdirectoryDescendants),
+          !contents.isEmpty else { return nil }
+    return contents.max { a, b in a.creationDate! < b.creationDate! }
```

Comment:
```suggestion
    return contents.filter { $0.creationDate != nil }.max { $0.creationDate! < $1.creationDate! }
```
Or if you are sure that `creationDate` would never be `nil`, you can remove the `filter` clause.

Reply:
Fixed. Rebased with develop. Tested again. I kept the filter. Made me worry that someone could put the screen shot directory on some network device that doesn't provide it.